.github/workflows/deploy-docs.yml

```
@@ -23,175 +21,67 @@ permissions:
-            🧪 This sticky comment auto-updates on every push.
-            📦 Source: branch `${{ github.ref_name }}` @ `${{ github.sha }}`
-            🧭 Trigger: `${{ github.event_name }}`
\ No newline at end of file
+          append: "true"
+          hide_details: "true"
+
+          # Vercel
+          vercel_token: ${{ secrets.VERCEL_TOKEN }}
+          vercel_project_id: ${{ secrets.VERCEL_PROJECT_ID }}
+          vercel_org_id: ${{ secrets.VERCEL_ORG_ID }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)